#!/usr/bin/env python3
PACKAGE = "simple_sim_h_working"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("thresh",                  int_t,    0, "Thresh Value", 250, 0, 255)
gen.add("contour",                  int_t,    0, "Maximum Contour Size", 20000, 0, 20000)
gen.add("speed",                  double_t,    0, "Speed", 1.0, 0.0, 25.0)
gen.add("enable_follow",            bool_t,   0, "Enable",       False)
gen.add("height", double_t, 0, "Height", 0.6, 0, 1.0)

gen.add("median_blur",                  int_t,    0, "Median Blur", 7, 0, 47)
gen.add("ellipse",                  int_t,    0, "Eliipse", 5, 0, 30)
gen.add("gray",                  int_t,    0, "Gray", 5, 0, 30)
gen.add("turn",                  double_t,    0, "Turn", 1.0, 0, 2.0)
gen.add("middle",                  int_t,    0, "Middle", 600, 0, 1000)
exit(gen.generate(PACKAGE, "follow_lane", "Follow"))

